src/partials/global/_navigation.html

```diff
@@ -0,0 +1,60 @@
+<div class="dms-band rh-design-nav-shelf py-0 border-right">
+    <div class="nav-container pb-4">
+        <div id="rhDesignNav" class="accordion">
+            <div class="nav-group-wrapper nav-button design-nav-item mt-2">
+                <a href="/" class="pl-4 py-2">
+                   Overview
+                </a>
+            </div>
+            <div class="nav-group-wrapper">
+                <button href="//design.redhat.com/" id="stories-btn" class="accordion-toggle collapsed nav-button py-2 pl-4 mt-2" data-toggle="collapse" data-target="#stories">
+                    <p>Designer stories</p>
+                </button>
+                <div id="stories" class="accordion-panel collapse pr-0" data-parent="#rhDesignNav">
+                    <ol class="mb-0 pl-2 pb-1">
+                        <li class="mb-2 story design-nav-item">
+                            <a href="/designer-stories" title="All stories" class="py-1 pr-12">All stories&nbsp;</a>
+                        </li>
+
+                        <li class="mb-2 story design-nav-item">
+                            <a href="/designer-stories/allona-wilkerson.html" title="Allona Wilkerson" class="py-1 pr-7">Allona Wilkerson</a>
+                        </li>
+
+                        <li class="mb-2 story design-nav-item">
+                            <a href="/designer-stories/ann-nguyen.html" title="Ann Nguyen" class="py-1 pr-10">Ann Nguyen&nbsp;</a>
```

Comment:
Is the `&nbsp;` here necessary?

Reply:
Yep. Using to avoid custom css for shorter names